runtime/gc_verbose_handler_vlhgc/VerboseHandlerOutputVLHGC.hpp

```diff
@@ -156,7 +156,6 @@ class MM_VerboseHandlerOutputVLHGC : public MM_VerboseHandlerOutput
 	
 	virtual	void handleConcurrentStartInternal(J9HookInterface** hook, UDATA eventNum, void* eventData);
 	virtual void handleConcurrentEndInternal(J9HookInterface** hook, UDATA eventNum, void* eventData);
-	virtual const char *getConcurrentTypeString() { return "GMP work packet processing"; }
```

Comment:
With this removed, there's a complication on relying on `getCycleType(_stats->concurrentType)` for the concurrent type string. 
`OMR_GC_CYCLE_TYPE_VLHGC_GLOBAL_MARK_PHASE` will result in "global mark phase" (see MM_VerboseHandlerOutputVLHGC::getCycleType)  and according to this original implementation it should be "GMP work packet processing"